inc/wpseo-non-ajax-functions.php

```diff
@@ -573,3 +358,61 @@ function wpseo_allow_system_file_edit() {
 
 	return WPSEO_Utils::allow_system_file_edit();
 }
+
+/**
+ * Test whether force rewrite should be enabled or not.
+ *
+ * @deprecated 3.3
+ *
+ * @return void
+ */
+function wpseo_title_test() {
+	return;
```

Comment:
Is there a reason for the lonely return here or do you think it's just nicer? Would an empty function block not work? (Ignoring my `_deprecated_function` comment for a moment)